angular.json

```diff
@@ -65,7 +65,8 @@
             "styles": [
               "projects/demo/src/styles.css"
             ],
-            "scripts": []
+            "scripts": [],
+            "es5BrowserSupport": true
```

Comment:
Should not be necessary

Reply:
Right. I added it in the first commit, and forgot to remove it in the second one.